ukpa/templates/ukpa/_nomination_form.html

```diff
@@ -0,0 +1,31 @@
+{% load bootstrap3 %}
+{% load avatar_tags %}
+<p>Before submitting this form, you should ensure that you have read the <a href="http://2017.pyconuk.org/ukpa/trustess/">Trustee Guidelines</a> on the main conference website.
+<form method="post">
+  {% csrf_token %}
+
+  <table class="table borderless">
+    <tr>
+      <td>{% avatar user %}</td>
```

Comment:
the available options are described at https://en.gravatar.com/site/implement/images/#default-image

I've changed it to 'mm' - 'Mystery Man'